daemons/compute/client-cpp/Readme.md

```diff
@@ -5,35 +5,44 @@ A basic example of a data movement client written in C++
 The Makefile, CMakeLists.txt, and common.cmake files are modifications to the [helloworld cpp example](https://github.com/grpc/grpc/tree/v1.46.3/examples/cpp/helloworld).
 
 ## Example code and setup
+
 1. Follow the Quick start instructions to [build and locally install gRPC from source](https://grpc.io/docs/languages/cpp/quickstart/#install-grpc)
-2. Run `cmake`
+2. Run `cmake` in this directory (i.e. `client-cpp`):
 
+```shell
+mkdir -p cmake/build
+cd cmake/build
+cmake -DCMAKE_PREFIX_PATH=$MY_INSTALL_DIR ../..
 ```
-$ mkdir -p cmake/build
-$ cd cmake/build
-$ cmake -DCMAKE_PREFIX_PATH=$MY_INSTALL_DIR ../..
-```
+
 ### Generating client code
```

Comment:
Maybe a section with `make protoc` to update the generated API code

Reply:
The cmake/build/Makefile does this for us, so I don't think there's a need. Should I just remove the Makefile in client-cpp?

Reply:
Yeah, that would avoid future headaches.